src/main/java/org/ssor/boss/core/exception/UserDataAccessException.java

```diff
@@ -0,0 +1,12 @@
+package org.ssor.boss.core.exception;
+
+public class UserDataAccessException extends RuntimeException {

```

Comment:
@christianangeles Bracer here should be new line
```suggestion
public class UserDataAccessException extends RuntimeException 
{

```